lib/webpacker/dev_server_runner.rb

```diff
@@ -50,7 +50,7 @@ def execute_cmd
         cmd = if node_modules_bin_exist?
           ["#{@node_modules_bin_path}/webpack-dev-server"]
         else
-          ["yarn", "webpack-dev-server"]
+          ["npx", "webpack-dev-server"]
         end
 
         if @argv.include?("--debug-webpacker")
```

Comment:
@dcwither hi there! I noticed that `npm run webpack-watch` in traject doesn't show the compilation progress stats you'd expect to see through the `--progress` arg [being passed in here](https://github.com/FB-PLP/traject/blob/master/package.json#L282). I _think_ this is happening because `@argv` here isn't actually passed into the `exec` command a few lines below like it is [in the original repo](https://github.com/rails/webpacker/blob/master/lib/webpacker/dev_server_runner.rb#L82). Guessing that also means we're not passing in `--devtool eval` things to WDS.

The `@argv` addition to the final command seems to have been introduced [here](https://github.com/rails/webpacker/commit/2b17c3fb72e16677e7dc4061a822a4efc08f9aa4). Any way we could update the fork we're using? Alternatively, I'm happy to patch-add just the one line.

Reply:
Feel free to make a PR to this fork with those changes and I can review it. The `--devtool eval` piece may help with performance if we're not already doing that in our configs.

Ultimately we want to move fully to yarn since rails doesn't support `npm`